nombre

NoCodeRights opened the pull request:
agregué mi nombre a la página, para ser inmortalizado en el código html de la máquina. Praise the Omnissiah.

Gh0stRaccoon reviewed (changes requested):
Porfavor agrega tu nombre al index.html

NoCodeRights committed:
nombre actualizado

modifiqué directamente en la página. Puede explicar cómo modificar a través de visual code por favor? creo que me enredé

Gh0stRaccoon reviewed (approved):
🚀

Gh0stRaccoon committed:
Merge branch 'main' into marc-sep
